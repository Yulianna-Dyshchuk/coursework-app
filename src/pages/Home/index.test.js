import React from "react";
import { shallow } from "enzyme";
import { Home } from "./index";

describe("Home component", () => {
  it("should render correctly without props", () => {
    const component = shallow(<Home />);

    expect(component).toMatchSnapshot();
  });
});
