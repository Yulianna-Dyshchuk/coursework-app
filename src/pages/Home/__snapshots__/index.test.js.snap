// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render correctly without props 1`] = `
<main>
  <img
    alt="img"
    src="https://drive.google.com/uc?id=1NHSkvGSdcL8lpsARa4pLgP7F-5rMJFKh"
  />
</main>
`;
